src/main/resources/application.conf

```diff
@@ -47,6 +47,9 @@ mantis {
       port = 9076
     }
 
+    # Try automatic port forwarding via UPnP
+    automatic-port-forwarding = true
```

Comment:
Should this be on by default?

Reply:
For it to be useful, I think so. Someone who wouldn't map the ports themselves probably won't be inclined to enable this manually either, right?